File: AndroidStudioProjects/MultipleLanguageSupport/app/src/main/java/com/rashedul/multiplelanguagesupport/MainActivity.java
package com.rashedul.multiplelanguagesupport;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    TextView hello, night;
    Button button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        hello = findViewById(R.id.textView);
        night = findViewById(R.id.textView2);
        button = findViewById(R.id.button);

        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(getApplicationContext(),R.string.toastMessage,Toast.LENGTH_LONG).show();

            }
        });



    }
}

File: AndroidStudioProjects/Playing/app/src/main/java/com/rashedul/playing/MainActivity.kt
package com.rashedul.playing

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
//        setContentView(R.layout.activity_main)
        setContentView(R.layout.note_layout)
        val click: Button? = findViewById(R.id.ok)




    }
}